Resources/Prototypes/Entities/Clothing/Shoes/magboots.yml

```diff
@@ -28,6 +28,11 @@
     - type: Tag
       tags:
       - WhitelistChameleon
+    - type: ReverseEngineering # Nyano
```

Comment:
I'm not *really* sure if this should be tagged Nyano or Delta. Regardless, this should be tagged with a bit more context than just "Nyano" so it's *very* clear to whoever comes next that ReverseEngineering is specifically a Nyano system.

Reply:
yeah makes sense. ill update em